code/control/src/pure_pursuit_controller.py

```
@@ -7,6 +7,7 @@
 from nav_msgs.msg import Path
 from ros_compatibility.node import CompatibleNode
 from rospy import Publisher, Subscriber
+import rospy
```

Ruff (0.8.2)

Redefinition of unused `rospy` from line 5

Remove definition: `rospy`

(F811)